### PR TITLE
Honeybee scale Non-Uniformly and new functionality to Honeybee_addHBGlz

### PR DESCRIPTION
**Pull request by antonszilasi**
Added functionality to Honeybee_addHBGlz so that if existing fenestration surfaces are passed through  _childSurfaces the radiance material properties of those surfaces will be kept. I found this handy in certain workflows

### CODE REVIEW

**Comment by mostaphaRoudsari** [comment]
@antonszilasi thanks. You're missing the userobject for addHBGlz.

File: src/Honeybee_Scale Honeybee.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        
+    if not Z:
+        
+        Z = 1
+    
+    # create a NU scale
+    
+    NUscale = rc.Geometry.Transform.Scale(P,X,Y,Z)
+    
+    print NUscale
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Scale Honeybee.py]: Do you really need to print the scale?

**Comment by antonszilasi** [comment]
@mostaphaRoudsari the userobject hasnt been changed do it didnt update in the commit.

File: src/Honeybee_addHBGlz.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
 
 ghenv.Component.Name = 'Honeybee_addHBGlz'
 ghenv.Component.NickName = 'addHBGlz'
-ghenv.Component.Message = 'VER 0.0.59\nJAN_26_2016'
+ghenv.Component.Message = 'VER 0.0.59\nJUNE_11_2016'
```
Comment (mostaphaRoudsari) [inline src/Honeybee_addHBGlz.py]: This should be +ghenv.Component.Message = 'VER 0.0.59\nJUN_11_2016'. Remove the E at the end of the month.

**Comment by antonszilasi** [comment]
@mostaphaRoudsari  try this for now, I've emailed you the file with the updated addHBGlzing as the LadybugExport component is not working

**Comment by chriswmackey** [comment]
@antonszilasi ,

I agree that a scaling component is useful and necessary if we want to have all 4 of the mathematical transformations available to HBObjects (translation, rotation, reflection, and scaling).  However, I'm having a hard time understanding the final changes made to the code base from this pull request given your several commits.  I would suggest either one of the following:

Option 1 - Close this pull request, revert the changes to your branch, export a new scale NU component to your branch, and send a new pull request to the master with just the scale component.  I'll look it over and merge it in if all looks good.

Option 2 - Wait until you are in Boston and we'll just figure all of it out then.

-Chris

**Pull request closed without merging**